<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teamsListWindow</class>
 <widget class="QMainWindow" name="teamsListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teams List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="teamsList"/>
    </item>
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Show Team</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
